Shared/Samples/Download vector tiles to local cache/README.md

```diff
@@ -39,7 +39,7 @@ When the vector tiled layer loads, zoom in to the extent you want to export. The
 
 **NOTE:** Downloading tiles for offline use requires authentication with the web map's server. To use this sample, you will need an [ArcGIS Online](https://www.arcgis.com) account.
 
-Vector tiles have high drawing performance and smaller file size compared to regular tiled layers, due to consisting solely of points, lines, and polygons. However, in ArcGIS Maps SDK they cannot be displayed in scenes. Visit [ArcGIS for Developers](https://developers.arcgis.com/ios/layers/#layer-types) to learn more about the characteristics of ArcGIS vector tiled layers.
+Vector tiles have high drawing performance and smaller file size compared to regular tiled layers, due to consisting solely of points, lines, and polygons. However, in ArcGIS Maps SDK they cannot be displayed in scenes. Visit [ArcGIS for Developers](https://developers.arcgis.com/swift/layers/#layer-types) to learn more about the characteristics of ArcGIS vector tiled layers.
```

Comment:
```suggestion
Vector tiles have high drawing performance and smaller file size compared to regular tiled layers, due to consisting solely of points, lines, and polygons. However, in ArcGIS Maps SDKs for Native Apps they cannot be displayed in scenes. Visit [ArcGIS for Developers](https://developers.arcgis.com/swift/layers/#layer-types) to learn more about the characteristics of ArcGIS vector tiled layers.
```

Reply:
of just say "in this SDK"

Reply:
Hmm, upon reading this, I wonder if it still stands. I've asked question and will make follow-up updates if needed.

Reply:
Verified that the sentence is no longer valid, so I rephrased the paragraph.